source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx

```diff
@@ -33,7 +34,17 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
           );
 
         case EditFormNames.propertyDetails:
-          return <></>;
+          const propertyFile = getAcquisitionFileProperty(
+            props.acquisitionFile,
+            props.selectedMenuIndex,
+          );
+          return (
+            <UpdatePropertyDetailsContainer
+              id={propertyFile?.property?.id as number}
```

Comment:
nit, is the cohesion necessary?

Reply:
I can probably refactor this to get rid of this cast, yes.